src/models/encrypted-model.test.ts

```diff
@@ -44,12 +44,14 @@ Dummy.init({
 }, {
 	hooks: {
 		beforeSave: async (app, opts) => {
-			await app.encryptChangedSecretFields(opts.fields, getLogger("test"));
+			const optsFields = typeof opts.fields === "string" ? opts.fields : undefined;
```

Comment:
Turns out the fields property can hold a string, number, or symbol but in `encryptChangedSecretFields` we are setting `fieldsChanged` to `string[] = []`